### PR TITLE
One line one sentence for expressions and statements main chapters

### PR DESCRIPTION
**Pull request by Havvy**
Mechanical PR that converts the expressions chapter to having one sentence per line.

### CODE REVIEW

**Comment by Havvy** [comment]
I added the "statements" and "statements and expressions" chapters as well.

File: src/expressions.md [hunk 1]
```diff
@@ -134,23 +127,27 @@ assert_eq!(
...
+> **Note**: Since this is applied recursively, these expressions are also evaluated from innermost to outermost, ignoring siblings until there are no inner subexpressions.
+
+## Early Termination
+
+Expressions may be *terminated* early, whether by a jump expression or unwinding.
+When terminated, evaluation of the expression stops.
+If the expression is the target of termination by a jump expression, then it evaluates to the value specified by the jump expression.
+Otherwise, it evaluates to the never type.
+
+**Note**: Destructors are still executed for the expression after being terminated.
```
Comment (ehuss) [inline src/expressions.md:139]: Is this new content?  If so, can it be added in a separate PR?

Comment (Havvy) [reply src/expressions.md:139]: Yes, that was meant to not go along with it.

**Comment by Havvy** [comment]
I rebased out the extra stuff.

**Review by ehuss — APPROVED** [review]
Thanks!  I resolved the merge conflict.
